clients/lsp-rust.el

```diff
@@ -549,9 +554,43 @@ The command should include `--message=format=json` or similar option."
          (result (lsp-send-request (lsp-make-request "experimental/joinLines" params))))
     (lsp--apply-text-edits result)))
 
+(defcustom lsp-rust-analyzer-download-url
+  (format "https://github.com/rust-analyzer/rust-analyzer/releases/download/nightly/%s"
```

Comment:
Shouldn't this be `latest` instead of `nightly`? At least for the default.

Reply:
```suggestion
  (format "https://github.com/rust-analyzer/rust-analyzer/releases/latest/download/%s"
```

Reply:
@flodiebold any opinion on that? Is nighly stable enough for daily use or better to use the latest stable?

Reply:
I agree the weekly release is a better default.

Reply:
Thank you, applied the comment.